cloudify_cli/utils.py

```diff
@@ -344,8 +347,21 @@ def get_rest_client(manager_ip=None, rest_port=None, protocol=None):
 
     trust_all = get_ssl_trust_all()
 
-    return CloudifyClient(host=manager_ip, port=rest_port, protocol=protocol,
-                          headers=headers, cert=cert, trust_all=trust_all)
+    client = CloudifyClient(host=manager_ip, port=rest_port, protocol=protocol,
+                            headers=headers, cert=cert, trust_all=trust_all)
+
+    if skip_version_check:
+        return client
+
+    cli_version, manager_version = get_cli_manager_versions()
+
+    if cli_version == manager_version:
+        return client
+    else:
+        message = ('CLI and manager versions do not match\n'
+                   'Cloudify CLI: {}\n'
+                   'Cloudify Manager: {}').format(cli_version, manager_version)
```

Comment:
Right now, our convention is to keep compatibility with python 2.6 in any new code. While the CLI itself doesn't fully support 2.6, we should still keep that compat until decided otherwise. {} will fail on string formatting. please use {0}